rotkehlchen/api/rest.py

```diff
@@ -654,7 +654,7 @@ def get_supported_chains(self) -> Response:
             if blockchain == SupportedBlockchain.OPTIMISM:
                 data['native_asset'] = A_ETH.serialize()
             elif blockchain == SupportedBlockchain.POLYGON_POS:
-                data['native_asset'] = A_POLYGON_POS_MATIC.serialize()
+                data['native_asset'] = A_MATIC.serialize()
```

Comment:
Why this change? Shouldn't they be equivalent? And since POLYGON_POS_MATIC is the asset in the POS chain that should be the correct one.

Reply:
I thought about it and yeah, it sounds reasonable. Will change